LICENSE

```diff
@@ -1,201 +1,99 @@
+
+**2.1.5.	Additional Conditions.**  This License is not an assurance (i) that any of Contributor’s copyrights or issued patent claims cover an Implementation of the Specification or are enforceable or (ii) that an Implementation of the Specification would not infringe intellectual property rights of any third party.
+
+**2.2.	Patent Licensing Commitment.**  In addition to the rights granted in Section 2.1, Contributor agrees to grant everyone a no charge, royalty-free license on reasonable and non-discriminatory terms to Contributor’s Necessary Claims that are within Scope for:
+1) Implementations of a Draft Specification, where such license applies only to those Necessary Claims infringed by implementing Contributor's Contribution(s) included in that Draft Specification, and
+2) Implementations of the Approved Specification. 
+
+This patent licensing commitment does not apply to those claims subject to Contributor’s Exclusion Notice under Section 3.
+
+**2.3.	Effect of Withdrawal.**  Contributor may withdraw from the Working Group by issuing a pull request or commit providing notice of withdrawal to the Working Group repository’s Notices.md file.  All of Contributor’s existing commitments and obligations with respect to the Working Group up to the date of that withdrawal notice will remain in effect, but no new obligations will be incurred. 
+
+**2.4.	Binding Encumbrance.**  This License is binding on any future owner, assignee, or party who has been given the right to enforce any Necessary Claims against third parties.
+
+**3.	Patent Exclusion.**
+
+**3.1.	As a Result of Contributions.**  Contributor may exclude Necessary Claims from its licensing commitments incurred under Section 2.1.1 by issuing an Exclusion Notice within 45 days of the date of that Contribution.  Contributor may not issue an Exclusion Notice for any material that has been included in a Draft Deliverable for more than 45 days prior to the date of that Contribution.
+
+**3.2.	As a Result of a Draft Specification Becoming an Approved Specification.**  Prior to the adoption of a Draft Specification as an Approved Specification, Contributor may exclude Necessary Claims from its licensing commitments under this Agreement by issuing an Exclusion Notice.  Contributor may not issue an Exclusion Notice for patents that were eligible to have been excluded pursuant to Section 3.1.
+
+**4.	Source Code License.**  Any source code developed by the Working Group is solely subject the source code license included in the Working Group’s repository for that code.  If no source code license is included, the source code will be subject to the MIT License.
```

Comment:
Is this default correct or should it be Apache?

Reply:
This is correct, and copied from https://github.com/CommunitySpecification/Community_Specification/blob/main/1._Community_Specification_License-v1.md

The link with Apache is when it says `If no source code license is included, the source code will be subject to the MIT License.` , and that's why we have added mentions to Apache in the README.

If you think we should make this more prominent, we could add a `SOFTWARE-LICENSE` in the root folder, containing Apache 2.0 License.

Reply:
@maoo I didn't understand this sentence:

>The link with Apache is when it says `If no source code license is included, the source code will be subject to the MIT License.` , and that's why we have added mentions to Apache in the README.

How does this reference to the MIT license link to the Apache license?? I thought defaulting to the Apache license was the main change we were making to the CSL and that this is the place that its done...

Reply:
@robmoffat unresolved this as I'm still not sure its right - or I'm just not understanding something

Reply:
@kriswest - sorry, I admit that my explanation was not very clear/exhaustive ; I'll try again.

In order to keep things simple, we have decided to use CSL as-is, without any modification, for the following reasons:
1. It's a simpler approach, as we can point the FDC3 community to a known license, rather than a patched version of it - less docs, less questions, easier to integrate with other tools (ie SPDX, GitHub APIs, etc)
2. It can be used along with any other license, as long as it's stated in the project's README and/or Notices (MIT is just a fallback option)

Based on the second point, we made sure that the README.md (see section `License & Patents`) specifies that `Reference implementations and other software contained in FDC3 repositories is licensed under the Apache License, Version 2.0`.

We have validated this approach with our Legal team and we got their approval.

Reply:
Also we updated all of the headers in source files to be Apache